src/gvars.c

```diff
@@ -825,6 +881,37 @@ static Obj FuncIsReadOnlyGVar (
     return IsReadOnlyGVar(GVarName(CSTR_STRING(name))) ? True : False;
 }
 
+/****************************************************************************
+**
+*F  IsConstantGVar( <gvar> ) . . . . . . return if a variable is a constant
+*/
+Int IsConstantGVar(UInt gvar)
+{
+    return INT_INTOBJ(ELM_GVAR_LIST(WriteGVars, gvar)) == -1;
+}
+
+/****************************************************************************
+**
+*F  FuncIsConstantGVar( <name> ) . . .handler for GAP function
+**
+*/
+
+static Obj FuncIsConstantGVar (
+    Obj                 self,
+    Obj                 name )
+{
+    /* check the argument                                                  */
```

Comment:
Personally, I try to avoid those "right aligned with lots of spaces" comments, and just use C++ style `//` comments. While this makes the new code slightly different than the old code, I think we should eventually replace all those weird aligned comments with these

(That's just a vague suggestion, not an actual request to change the PR).